doc-content/enterprise-only/admin-and-config/setup-auto-cleanup-proc.adoc

```diff
@@ -4,16 +4,20 @@
 You can set up an automatic cleanup job in {CENTRAL}.
 
 .Procedure
-. In {CENTRAL}, go to *Deploy* > *Jobs*.
+. In {CENTRAL}, go to *Manage* > *Jobs*.
 . Click *New Job*.
-. Enter a name, due date, time, and the following command into the *Type* text field.
+. Enter a *Business Key*, *Due On*, *Retries*, and the following command into the *Type* field.
 +
 [source]
 ----
 org.jbpm.executor.commands.LogCleanupCommand
 ----
 
-. Click *Add Parameter* if you want to use the <<logcleanupcommand_parameters_{context}, parameters>>, enter a parameter in the key section and enter a parameter value in the value section.
+. If you want to use the <<logcleanupcommand_parameters_{context}, parameters>>:
++
+.. Open the *Advanced* tab.
+.. Click *Add Parameter*.
```

Comment:
Will be good to add a link to a full parameters list if exists or create a jira for this page.

Reply:
@hasys the cross-reference of the full parameter list is already available in the sentence above:

check sentence: If you want to use the <<logcleanupcommand_parameters_{context}, parameters>>:

Reply:
Sorry, I missed it. So, maybe it will be better to move the link to separate sentence like "For full list of parameters see this section" or something like this.

But it is optional. Thank you!

Reply:
Done